Dockerfile

```diff
@@ -1,7 +1,13 @@
 FROM openjdk:8-alpine
 
-COPY target/uberjar/chocolate.jar /chocolate/app.jar
+WORKDIR /chocolate/
+
+COPY target/uberjar/chocolate.jar app.jar
 
 EXPOSE 3000
 
-CMD ["java", "-jar", "/chocolate/app.jar"]
+ENV DATABASE_URL="jdbc:sqlite:./chocolate_dev.db"
```

Comment:
a few minor things.

1) How do we need to configure the local env to run the uberjar directly? "java -jar target/uberjar/chocolate.jar" doesn’t seem to work, I assume because it doesn’t know how to find the database. Can you add this to the readme?

2) deploying the docker file runs (localhost:3000 renders the web page), but it can’t find a rabbitmq broker that might be running on the dev machine because, I assume, rabbit isn’t visible from inside the container. Have you worked out how to deploy rabbit in there as well? or is there a way to let chocolate-conatiner see  rabbit that’s already running on the local host?

3) can you update the readme once the first 2 are sorted?

Reply:
I've updated the readme.  Indeed running the uberjar with just java -jar doesnt work, regardless of where the db file is located, it doesnt pickup any :database-url env variable, and therefore doesnt run.  It runs by itself in a container, but not sure how to configure it to talk to a local rabbitMQ instance. 

Using docker-compose with rabbitMQ included seems to work great however, give it a try!